pkg/rpc/response/result/block.go

```diff
@@ -48,7 +45,7 @@ type (
 
 		Script transaction.Witness `json:"script"`
 
-		Tx []Tx `json:"tx"`
+		Tx []*Tx `json:"tx"`
```

Comment:
`Tx` is quite simple, do we need a pointer here for some reason?

Reply:
No, we don't. Fixed